.github/workflows/my-ci-pipeline.yml

```
@@ -0,0 +1,19 @@
+name: Continuous Integration
+on: 
+  pull_request:
+    types: 
+      - opened
+    paths-ignore:
+      - 'README.md'
+  push:
+    paths-ignore:
+      - 'README.md'
+jobs:
+  build:
+    name: Build and test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - run: docker build --target test --tag todo-app:test .
+    - run: docker run --env-file .env.test --publish 127.0.0.1:5000:5000  todo-app:test todo_app/tests/test_view_model.py
+    - run: docker run --env-file .env.test --publish 127.0.0.1:5000:5000  todo-app:test todo_app/tests/test_Trello_Integration.py
```

No harm caused here by this, but since we're not running anything outside the container that wants to access the app, there's no need to publish the ports here, and you've already copied the `.env.test` file into the container, so we can skip that too!
```suggestion
    - run: docker run todo-app:test todo_app/tests/test_view_model.py
    - run: docker run todo-app:test todo_app/tests/test_Trello_Integration.py
```

Also, I'd argue that this isn't going to be very pleasant as the number of tests increases - if we end up with a few hundred test files, we'll definitely be grateful if we just started by having a single test command - so can we drop the specific file names at the end, and just let pytest run them all?
```yml
    - run: docker run todo-app:test
```